index.html

```diff
@@ -937,20 +937,32 @@ <h2>
         </p>
       </section>
     </section>
-    <section class="informative">
+    <section>
       <h2>
         Privacy and Security Considerations
       </h2>
-      <section>
-        <h3>
-          Access to aspects of a user's local computing environment
-        </h3>
-        <p>
-          The screen orientation type and angle of the device can be accessed
-          with the API specified in this document, and can be a potential
-          fingerprinting vector.
-        </p>
-      </section>
+      <p>
+        A screen's [=current orientation type|type=] and [=current orientation
+        angle|angle=] are a potential fingerprinting vector. To resist
+        fingerprinting (e.g., in private browsing), user agents MAY:
+      </p>
+      <ol>
+        <li>Restrict the value return by the {{ScreenOrientation/type}}
+        attribute to {{OrientationType/"portrait-primary"}} or
+        {{OrientationType/"landscape-secondary"}} to match the screen's aspect
+        ratio.
+        </li>
```

Comment:
This still reads a little weird. Can it return one of those or both, but what it returns depends on the aspect ratio? (Presumably the latter.)

Reply:
Maybe something along these lines:

> Restrict the possible return values of the `type` getter to portrait and landscape. The screen aspect ratio determines which is returned.

Reply:
used your suggestion.